salt/states/file.py

```diff
@@ -30,6 +59,11 @@
             custom_var: "override"
     {% endif %}
 
+.. _use-py-renderer:
+
+Use py renderer as a templating option
+--------------------------------------
+
 It is also possible to use the :mod:`py renderer <salt.renderers.py>` as a
```

Comment:
It is also possible to use _any_ renderer.

Reply:
Good catch, I'll make that change.